generators/example/templates/_index.html

```diff
@@ -0,0 +1,22 @@
+
+## It Works!
```

Comment:
@yoshokatana This file replaces `component/ hello-world/template.nunjucks`, correct?